docs/src/pages/roadmap.tsx

```diff
@@ -239,12 +232,26 @@ const roadmap: Item[] = [
 ];
 
 const milestones: Item[] = [
+  {
+    icon: mdiStar,
+    iconColor: 'gold',
+    title: '70,000 Stars',
+    description: 'Reached 70K Stars on GitHub!',
+    getDateLabel: withLanguage(new Date(2025, 6, 9)),
```

Comment:
Should that be
```suggestion
    getDateLabel: withLanguage(new Date(2025, 7, 9)),
```
?

Reply:
https://main.preview.immich.app/roadmap it's correct here :)